torch/csrc/jit/codegen/cuda/tensor_view.cpp

```diff
@@ -1158,6 +1159,34 @@ void TensorView::circularBuffer(unsigned int stage) {
   circular_buffer_stage_ = stage;
 }
 
+namespace {
+
+void validateSkewDoubleBuffer(TensorView* tv) {
```

Comment:
It wasn't clear until this point what the requirements of using `skewDoubleBuffer` are. 

It seems that those requirements are:

- The tv itself is already set to be double buffered
- The producer of the tv is also double buffered.

I guess we may also want to validate more, for example, making sure the producer tensor is not skewed, as I assume this skewed double buffering won't work in that case.